lib/manager/components/NotesViewerPanel.js

```diff
@@ -33,16 +32,29 @@ export default class NotesViewerPanel extends Component {
     return null
   }
 
+  _toggleExpanded = () => {
+    const {notes, notesRequested} = this.props
+    const {expanded} = this.state
+    if (!notes) notesRequested()
+    this.setState({ expanded: !expanded })
+  }
+
   render () {
     const messages = getComponentMessages('NotesViewer')
     const type = this.props.type === 'feed-source'
       ? getMessage(messages, 'feedSource')
       : getMessage(messages, 'feedVersion')
     const header = (
-      <h3 onClick={() => {
-        if (!this.props.notes) this.props.notesRequested()
-        this.setState({ expanded: !this.state.expanded })
-      }}>
+      <h3
+        onClick={this._toggleExpanded}
+        onKeyDown={(e) => {
+          if (document.activeElement === e.target && e.which === 13) {
+            this._toggleExpanded()
+          }
+        }}
+        role='button'
+        tabIndex={0}
```

Comment:
Oh boy, NotesViewerPanel is old code... isn't even imported anywhere.  Deleting.